core/src/main/java/de/danoeh/antennapod/core/feed/Note.java

```diff
@@ -0,0 +1,77 @@
+package de.danoeh.antennapod.core.feed;
+
+import android.database.Cursor;
+import android.util.Log;
+import androidx.annotation.NonNull;
+import de.danoeh.antennapod.core.storage.PodDBAdapter;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+
+import java.io.Serializable;
+
+public class Note implements Serializable {
```

Comment:
Yes thats also possible, but for me, losing the notes after intentionally (or accidentally) clearing the episodes was a concern, especially if I have not backed them yet into my actual Notes app. Either is fine.

Reply:
The screen shown when deleting a podcast could include a warning that it will also delete custom notes. You currently have to store the title of the episode again that a note refers to - that is more or less redundant when we can also store the notes directly with the episode.

By the way - sorry for taking so long to respond. This is a feature where I am not completely sure about how to integrate it, so thinking about it takes more brain power than the other PRs :D I must admit that I only looked at the easier PRs in the last few days.